silex/init.lua

```diff
@@ -7,19 +7,21 @@
 -- Some are fixes or workarounds for issues in SILE.
 --
 SILE.X = SILE.X or {
-  version = "0.6.0",
+   version = "0.6.0",
```

Comment:
I propose a major bump to this (whether that is 1.x or 0.7.x) that drops all attempts to backport functionality to older SILE versions. So far I haven't removed anything that is still a hack, workaround, or experiment in relation to the current SILE release. There are a few things left that ARE those things, but 95% of what I found so far is stuff that has already been upstreamed and was pre-merge experiments or backports to older SILE releases.

If supporting new packages on old SILE releases is a thing then we need to come up with a backports package or something that does explicitly just that without trying to drag thousands of lines of duplicated code into the dependency tree for projects trying to use *current* SILE releases.